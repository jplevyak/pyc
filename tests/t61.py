a = "foo %d bar %d" % (3,4)
print a
b = (1, 1.4, "hi")
print "foo %d bar %f wow %s" % b
d = 1.4
print "foo %f" % d
