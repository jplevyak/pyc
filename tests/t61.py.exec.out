foo 3 bar 4
foo 1 bar 1.400000 wow hi
foo 1.400000
